management /commands/datascript.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand, CommandError
+from entry.models import Entry
+
+class Command(BaseCommand):
+    help = "Creates superuser, creates non-superuser, creates 3 blog posts from non-super user"
+
+    def handle(self, *args, **options):
+        self.superuser()
+        self.non_superuser()
+
+    def superuser(self):
+        obj, created = Person.objects.get_or_create(superuser='tristin-admin')
+
+    def non_superuser(self):
+        obj, created = Person.objects.get_or_create(
+            superuser='tristin',
+            defaults={'blog 1': date(2015, 5, 20),
+                      'blog 2': date(2015, 5, 30),
+                      'blog 3': date(2015, 5, 25)})
```

Comment:
This all needs to be handled in two separate cases.  Keep this method's purpose to just create the "Person" object.  Then create a new method that accepts a user parameter and creates three blog entries for that user.  Something like:

```python
def non_superuser(self):
    # this method should only contain fields about the user.  So your defaults should 
    # be a dict of user fields only.  The key is the field, the value is the field value.
    return Person.objects.get_or_create(...)

def create_blog_entries(self, user):
    """The user is the user to create blog entries for."""
    # create 3 blog entries for the user here Blog.objects.create(...)
```